### PR TITLE
respect fstype in Windows for azure disk

### LINKED ISSUE #61266: respect fstype in Windows for azure disk
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

> Uncomment only one, leave it on its own line: 
>
> /kind bug
> /kind feature

**What happened**:
Currently azure disk mount on Windows only supports NTFS, we should respect fstype in Windows like what it does in Linux

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): v1.9 - v1.11
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

/sig azure
/sig windows
/assign

### PR DESCRIPTION
**Pull request by andyzhangx**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #61266

**Special notes for your reviewer**:

**Release note**:

```
respect fstype in Windows for azure disk
```

/sig azure
/sig windows
/assign @feiskyer

### CODE REVIEW

**Comment by andyzhangx** [comment]
@feiskyer

File: pkg/volume/azure_dd/azure_common_windows.go [hunk 1]
```diff
@@ -103,8 +103,12 @@ func formatIfNotFormatted(disk string, fstype string, exec mount.Exec) {
 		return
 	}
 
+	if len(fstype) == 0 {
+		// Use 'NTFS' as the default
+		fstype = "NTFS"
+	}
 	cmd := fmt.Sprintf("Get-Disk -Number %s | Where partitionstyle -eq 'raw' | Initialize-Disk -PartitionStyle MBR -PassThru", disk)
-	cmd += " | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume -FileSystem NTFS -Confirm:$false"
+	cmd += fmt.Sprintf(" | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume -FileSystem %s -Confirm:$false", fstype)
```
Comment (feiskyer) [inline pkg/volume/azure_dd/azure_common_windows.go]: Which filesystems are supported now on Windows? Should we validate the type as well?

Comment (andyzhangx) [reply pkg/volume/azure_dd/azure_common_windows.go]: @feiskyer There are NFTS, FAT32, REFS, etc. I would like to let user decide fstype, then in the future, if a new fstype is supported on Windows, we don't need to do any code change, same logic to Linux, it does not do any validation either.

**Review by feiskyer — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**